### ISSUE TITLE
test(a11y): Different text for alternative and caption

### ISSUE THREAD

**Issue by mfranzke** [db-ui/core#696]
Currently we're using the same text phrase for the alternative text of the image as well as the caption. These should be different as the information would be redundant (especially for screenreader users).

https://db-ui.github.io/core/patterns/elements-image-image-with-caption/elements-image-image-with-caption.rendered.html
